license

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
Rename this to LICENSE.md and put three.js under the third-party section as discussed above.

Let me look into CLAs before we select a license for our code, but I'll most likely suggest MIT.